logging/Readme.md

```diff
@@ -0,0 +1,35 @@
+# Logging
+
+Idiomatic Rust logging implementation.
+
+This PR is related to [holochain/lib3h#29](https://github.com/holochain/lib3h/issues/29) and implements a more idiomatic logging.
+
+Requirements can be found [here](https://hackmd.io/MP5F3UhSTp2iPk37Cwa-fw).
+
+It still misses:
```

Comment:
I think we can remove this from the readme.  It's good enough now.

Reply:
+1

Reply:
+1